src/components/signoff/SimpleReview/index.tsx

```diff
@@ -65,9 +65,25 @@ export default function SimpleReview({
   const destBid = signoffDest?.bid;
   const destCid = signoffDest?.cid;
 
+  let toReviewEnabled =
+    session.serverInfo?.capabilities?.signer?.to_review_enabled === true;
+  if (toReviewEnabled) {
+    const resourceMatch =
+      session.serverInfo?.capabilities?.signer?.resources?.find(
+        x =>
+          x.source.bucket === sourceBid &&
+          x.source.collection === sourceCid &&
+          x.destination.bucket === destBid &&
+          x.destination.collection === destCid
+      );
+    toReviewEnabled =
+      !resourceMatch || resourceMatch.to_review_enabled !== false;
+  }
+
```

Comment:
This could be moved into https://github.com/Kinto/kinto-admin/blob/master/src/components/signoff/utils.ts instead of duplicating